SmilesMiles/ViewModel/TripDetailViewModel.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  TripDetailViewModel.swift
+//  SmilesMiles
+//
+//  Created by Pavan Sai Nallagoni on 2/27/23.
+//
+
+import SwiftUI
+
+class TripDetailViewModel: ObservableObject {
+    @Binding var trips: [String]
```

Comment:
Modified. However, I'm planning to use Model to handle all the data store. For now, I've it as Published for each View Model so that I can move these to Model and maintain a sync between the data.